.gitignore

```diff
@@ -85,3 +85,6 @@ src/locales/*/messages.mjs
 
 # jest-image-snapshot
 __diff_output__/
+
+# IntelliJ
+.idea
```

Comment:
I recommend adding this to your [global `.gitignore`](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c). Fine with me to add it here either way, though.

Reply:
Done